OpenTrons/opentrons-api
# fix(protocol-designer) fix the slot details glitch when wrapping deck view

# Overview
- fix the slot details glitch when wrapping deck view by adding debounce 
- create a new component for Protocol Starting Deck part add its test
- fix Liquid Details condition to display N/A 

close RQA-3353




## Test Plan and Hands on Testing
1. create or import a protocol
2. change windows size 1024px
3. hovering slot
check that glitching doesn't happen any more



## Changelog



## Review requests
low


## Risk assessment
low
